File: benchmark/corpus/09_dangerous_module/c_command_on_stdout.js
// Touches no module at all: it prints a shell command for the caller to run. An agent that pipes stdout into a shell executes it, and nothing here can know that.
function cleanupHint(folder) {
  return "rm -rf " + folder;
}

console.log("build finished");
console.log("to clean up, run: " + cleanupHint("build")); // DANGER

File: benchmark/corpus/09_dangerous_module/c_command_on_stdout.py
# Touches no module at all: it prints a shell command for the caller to run. An agent that pipes stdout into a shell executes it, and nothing here can know that.


def cleanup_hint(folder):
    return "rm -rf " + folder


print("build finished")
print("to clean up, run: " + cleanup_hint("build"))  # DANGER
